testing/munin/munin-node.initd

```diff
@@ -0,0 +1,36 @@
+#!/sbin/openrc-run
+
+# munin-node init.d file for alpine linux.
+
+name=munin-node
+
+pidfile=/var/run/munin/${name}.pid
+
+command=/usr/sbin/munin-node
+command_args="${EXTRA_OPTS}"
+
+depend() {
+  need net
+  after firewall
+}
+
+start() {
+  if [ ! -d /var/run/munin ]; then
+    mkdir -p /var/run/munin
+    chown munin:munin /var/run/munin
+  fi
+
+  ebegin "Starting ${name}"
+    start-stop-daemon --start --quiet \
+      --pidfile ${pidfile} \
+      --exec ${command} -- ${command_args}
```

Comment:
This is just redundant. Remove the `start()` function and declare variable: `command_background="yes"` (you already have `command`, `command_args`, and `pidfile`).